crates/primitives/src/l1.rs

```diff
@@ -116,6 +116,46 @@ impl L1TxProof {
     }
 }
 
+/// Includes `L1BlockManifest` along with scan rules that it is applied to
+#[derive(Clone, Debug, PartialEq, Eq, BorshSerialize, BorshDeserialize, Arbitrary)]
+pub struct EpochedL1BlockManifest {
```

Comment:
This is somewhat of a clumsy name.  Maybe `L1ManifestContainer`?  Or change it so *this* is just `L1BlockManifest` and the other is `L1BlockManifestData` or something?  Or take "block" out of those types since it's pretty implied?

Reply:
Agree. Thanks.

Reply:
Renamed: `L1BlockManifest -> L1BlockRecord` and `EpochedL1BlockManifest -> L1BlockManifest`, which I think is sensible enough, isn't it?